twistr/events.go

```diff
@@ -532,6 +568,10 @@ func PlayCubanMissileCrisis(s *State, player Aff) {
 	   opponent will lose the game. This card’s Event may be canceled, at any time,
 	   if the USSR removes 2 Influence from Cuba or the US removes 2 Influence from
```

Comment:
We need to file a task for implementing this aspect of the card:
"This card's event may be canceled, at any time, if the USSR removes 2 influence from Cuba or the US removes 2 influence from either West Germany or Turkey"

Didn't know what that meant before. Reading that properly now, the player can choose to sacrifice the two influence at any time in the turn to cancel the event.

From twilight strategy:
"The influence removal is not an Action and can be done at any time. (It is unclear whether you can actually do this in the middle of your opponent’s Action, i.e., after he announces an intent to coup or realign Cuba, but this is probably an area better suited for sportsmanship than explicit timing rules.)"